app/routes/downloaddetails.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, ResponseToolkit, ResponseObject } from "@hapi/hapi";
+import { getDownloadDetailsCsv } from "../backend/api";
+
+module.exports = {
+  method: 'GET',
+  path: '/downloaddetails',
+  handler: async (request: Request, h: ResponseToolkit) => {
```

Comment:
Would it not make sense to add validation for the query parameters like we did with paymentdata endpoint?

Reply:
I skipped it because, the value comes from the internal code, not from the user. Anyway added the validation now.